src/gui/Base/HTrayIcon.qml

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+import Qt.labs.platform 1.1
+import Qt.labs.folderlistmodel 2.12
+
+SystemTrayIcon {
+    property string iconPack: theme ? theme.icons.preferredPack : "thin"
+    property alias settingsFolder: showUpWatcher.folder
+
+    property var window
```

Comment:
"Required" properties should be at the beginning, also you should use `ApplicationWindow` for the type instead of `var`